api/handler/acl.go

```diff
@@ -69,13 +69,15 @@ const (
 	awsPermRead        amazonS3Permission = "READ"
 )
 
-// GranteeType is aws grantee permission type constants.
-type GranteeType string
```

Comment:
Can also be omitted, s3-gw is not something that is supposed to be reused package-by-package.

Reply:
Then there is no sense to export these things.